src/cli/subcommand/uninstall.rs

```diff
@@ -125,17 +125,21 @@ impl CommandExecute for Uninstall {
         let (_tx, rx) = signal_channel().await?;
 
         let res = plan.uninstall(rx).await;
-        if let Err(err) = res {
-            if let Some(expected) = err.expected() {
-                println!("{}", expected.red());
-                return Ok(ExitCode::FAILURE);
-            }
-            return Err(err)?;
+        match res {
+            Err(err @ NixInstallerError::ActionRevert(_)) => {
+                tracing::info!("Uninstallation complete, some errors encountered");
```

Comment:
I kinda think this should be a warning, but that probably doesn't matter that much when we immediately return the errors that were encountered.